plugins/dualGovernance/hooks/useCanCreateProposal.tsx

```diff
@@ -39,11 +36,30 @@ export function useCanCreateProposal() {
         address: PUB_DUAL_GOVERNANCE_PLUGIN_ADDRESS,
         abi: OptimisticTokenVotingPluginAbi,
         functionName: "dao",
-      },
+      }
       // TODO: This needs to be checking as well if address has the DAO permission to create props
     ],
   });
 
+  // Check if PROPOSER_PERMISSION is granted to the current wallet
+  const { data: hasCreatePermission, refetch: hasCreatePermissionRefetch } = useReadContracts({
```

Comment:
The default value of the old useState() was `false`. Do we know what `hasCreatePermission` will be by until a result is received?
Not sure if `undefined`/`null` might bring some edge case

Reply:
Right now is not a problem because we are only using the negated value (!hasCreatePermission). But it's true it can be dangerous if we try to use the positive/raw value in the future. I will recover the boolean state functionality to be safe. Nice catch!